slider/library.py

```diff
@@ -187,7 +189,10 @@ def create_db(cls,
 
                 try:
                     beatmap = Beatmap.parse(data.decode('utf-8-sig'))
-                except ValueError as e:
+                except Exception as e:
+                    if skip_exceptions:
```

Comment:
Is this the first occurrence of logging in slider? I like the idea of logging here, but I am not sure it is worth cluttering user's logs.

Reply:
I think it is the first usage. I think that the user should know what beatmaps failed and why, so I don't see a good alternative.

Reply:
you could argue that because the user has explicitly passed a flag to skip exceptions, they don't want to see the log either. Personally I think it's good to log errors even with the flag passed.